stacks-signer/src/signerdb.rs

```diff
@@ -947,6 +979,23 @@ impl SignerDb {
         Ok(Some(receive_time))
     }
 
+    /// Get timestamp (epoch seconds) at which a burn block was received over the event dispatcheer by this signer
```

Comment:
```suggestion
    /// Get timestamp (epoch seconds) at which a burn block was received over the event dispatcher by this signer
```

Reply:
Fixed (as well as the comment this was copypasta from) in cac3a32